zerver/openapi/zulip.yaml

```diff
@@ -8474,6 +8474,233 @@ paths:
                               type: string
                               description: |
                                 The text content of the status message.
+                      user_settings:
+                        type: object
+                        description: |
+                          Present if `user_settings` is present in `fetch_event_types`.
+
+                          A dictionary which contains the settings of a user.
+
+                          **Changes**: New in Zulip 5.0 (feature level 89).
+                        additionalProperties: false
+                        properties:
+                          twenty_four_hour_time:
+                            type: boolean
+                            description: |
+                              Whether time should be [displayed in 24-hour notation](/help/change-the-time-format).
```

Comment:
I'm not reviewing the individual settings text, under the assumption that it's just moved.

Reply:
Yes, have copied it. I was not able to figure out how to avoid copying, especially with the `Changes` entries here in the `/settings` endpoint documentation.

Reply:
Yeah, the `Changes` system often ends up forcing copying, which is annoying, but not much to be done about it.